tool/sel4coreplat/sysxml.py

```diff
@@ -317,6 +318,7 @@ def xml2pd(pd_xml: ET.Element, plat_desc: PlatformDescription, is_child: bool=Fa
 
     pp = str_to_bool(pd_xml.attrib.get("pp", "false"))
     passive = str_to_bool(pd_xml.attrib.get("passive", "false"))
+    smc = str_to_bool(pd_xml.attrib.get("smc", "false"))
```

Comment:
Need to check that the architecture of the platform is AArch64, `smc` should not be able to be set otherwise.

Reply:
Actually we need to go one step further and check that `CONFIG_ALLOW_SMC_CALL` is enabled.